src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,8 +24,8 @@ import (
 @EmptyLine
 
 const (
-    major = "3"
-    minor = "1"
+    major = "4"
```

Comment:
Are you referring to PackageVersion option in command line. I have not found any [here](https://github.com/Azure/autorest/blob/master/Documentation/cli.md). Do you know how to use it?

`AutoRest.exe -CodeGenerator CSharp -Modeler Swagger -Input swagger.json -Namespace MyNamespace`